extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -51,7 +51,7 @@ describes.realWin('amp-form', {
 
   function getAmpForm(button1 = true, button2 = false, button3 = false,
                       canonical = 'https://example.com/amps.html') {
-    installAmpForm(env.win);
+    new AmpFormService(env.ampdoc);
```

Comment:
I see in [testing/describes.js](https://github.com/cvializ/amphtml/blob/master/testing/describes.js#L80) that `ampdoc` can have the following values: `"single", "shadow", "multi", "none"`. I don't see a `'fie'` option. Does that correspond to `'multi'`?